src/chrome/chrome.js

```diff
@@ -23,35 +23,37 @@ define(['jquery', 'windows/windows', 'moment', 'common/util'], function($, windo
         var show = check();
         if (show && chrome && chrome.webstore && $('#webtrader-extension-is-installed').length <= 0) {
             if (!win) {
-                win = windows.createBlankWindow($('<div class="chrome_extension"/>'),
+                require(['text!chrome/chrome.html'], function($html){
+                    $html = $($html);
+                    win = windows.createBlankWindow($html,
                     {
-                        title: 'Chrome Extension'.i18n(),
+                        dialogClass: "dialog-confirm",
                         width: 350,
-                        height: 200,
+                        height: 150,
                         resizable: false,
                         collapsable: false,
                         minimizable: false,
                         maximizable: false,
                         modal: true,
                         ignoreTileAction:true,
-                        'data-authorized': 'true',
-                        buttons: {
-                            Apply: function() {
-                                $( this ).dialog( 'close' );
-                                chrome.webstore.install();
-                                local_storage.set("chrome", { accepted_or_cancel_time : moment.utc().valueOf() });
-                            },
-                            Cancel: function() {
-                                $( this ).dialog( 'close' );
-                                local_storage.set("chrome", { accepted_or_cancel_time : moment.utc().valueOf() });
-                            }
-                        }
+                        'data-authorized': 'true'
```

Comment:
@4p00rv Please delete this line. Its not needed. This dialog is not required to be under authenticated context